crates/diesel_models/src/payment_method.rs

```diff
@@ -247,6 +251,7 @@ pub enum PaymentMethodUpdate {
     },
     ConnectorMandateDetailsUpdate {
         connector_mandate_details: Option<serde_json::Value>,
+        transaction_flow: Option<storage_enums::PaymentDirection>,
```

Comment:
Why are we coupling connector_mandate_details with the new Payouts enum?
Can you add more description for this change?

Reply:
as we are using connector_mandate_details for storing payout related data also, to identify the flow we are coupling it with transaction_flow

Reply:
we don't need to add `transaction_flow` during update. it was added by mistake. removed it